cmd/influxd/run/server.go

```diff
@@ -382,6 +382,9 @@ func (s *Server) Open() error {
 			go s.startServerReporting()
 		}
 
+		// Watch the meta store for a remote shutdown
+		go s.executeShutdown(s.MetaStore.ExecuteShutdown())
```

Comment:
Can you name this `monitorShutdown`? The name makes it sounds like it's actually going to start the shutdown process.